frontend-javascript-loops-master/loops-and-arrays.js

```diff
@@ -57,5 +74,13 @@ const squares = [30, 2, 8, 24, 11];
 //   'Het volume van 11 is 1331'
 // ]
 
+console.log(squares);
+
+for (let i = 0; i < squares.length; i++) {
+    const volume = squares[i] * squares[i] * squares[i];
```

Comment:
Wist je dat 
```suggestion
    const volume = squares[i] ** 3;
```
ook kan? 😉